slsReceiverSoftware/src/MasterAttributes.h

```diff
@@ -0,0 +1,567 @@
+#pragma once
+
+#include "ToString.h"
+#include "logger.h"
+#include "sls_detector_defs.h"
+
+#ifdef HDF5C
+#include "H5Cpp.h"
+#ifndef H5_NO_NAMESPACE
+using namespace H5;
+#endif
+#endif
+
+#include <chrono>
+using ns = std::chrono::nanoseconds;
+
+// versions
+#define HDF5_WRITER_VERSION   (6.1) // 1 decimal places
+#define BINARY_WRITER_VERSION (6.1) // 1 decimal places
+
+class MasterAttributes {
+  public:
```

Comment:
If all members are public I would declare this as a struct instead of a class.

Reply:
ok I just renamed the class to struct.

Reply:
done